toolchain/driver/driver.cpp

```diff
@@ -236,6 +241,12 @@ auto Driver::RunDumpSubcommand(DiagnosticConsumer& consumer,
                   /*IsForDebug=*/true);
   }
 
+  if (dump_mode == DumpMode::ObjectCode) {
+    CodeGen code_gen = CodeGen();
```

Comment:
Other comments of mine suggest getting rid of this class, but note that explicitly calling the constructor isn't needed.